src/tools/pbs_python.c

```diff
@@ -2310,11 +2310,37 @@ main(int argc, char *argv[], char *envp[])
 		char *largv[3];
 		int ll;
 		char *pc, *pc2;
```

Comment:
My comment was one of consistency.  Why do you go through the trouble of NULLing pc and pc2 (and setting ll=0) if you don't initialize them to begin with.  Neither of these steps are required with the way you are using them.  I'm just curious why you do one and not the other.